src/Core/src/Handlers/ViewHandlerExtensions.Android.cs

```diff
@@ -1,11 +1,68 @@
 ﻿using Android.Views;
 using Microsoft.Maui.Graphics;
 using PlatformView = Android.Views.View;
+using Microsoft.Maui.Platform;
+using System;
+using static Android.Views.View;
 
 namespace Microsoft.Maui
 {
-	public static partial class ViewHandlerExtensions
+	internal static partial class ViewHandlerExtensions
 	{
+		internal static void LayoutVirtualView(
```

Comment:
Copied this code out of each platforms `LayoutView`. I put these into extension methods so I can reuse them everywhere where I'm hosting an IView inside a platform control

Reply:
Probably worth creating a reminder to review these and the existing LayoutView/LayoutViewGroup/LayoutPanel stuff to see if we can consolidate it all into one spot.